Test/ContentPipeline/IntermediateDeserializerTest.cs

```diff
@@ -404,13 +404,15 @@ public void ExternalReferences()
         {
             Deserialize<ExternalReferences>("17_ExternalReferences.xml", externalReferences =>
             {
+                var pathSeparator = Path.DirectorySeparatorChar.ToString();
+
                 Assert.NotNull(externalReferences.Texture);
-                Assert.IsTrue(externalReferences.Texture.Filename.EndsWith(@"\Xml\grass.tga"));
+                Assert.IsTrue(externalReferences.Texture.Filename.EndsWith(pathSeparator + "Xml" + pathSeparator + "grass.tga"));
```

Comment:
I am confused by what you mean by that... if you mean the "17_ExternalReferences.xml" xml file, that file contains file name, not file path...